external_services/models.py

```diff
@@ -209,7 +209,7 @@ class MenuItem(UrlMixin, models.Model):
         max_length=32,
         null=True,
         blank=True,
-        help_text=_("Menu icon style name (else service default), e.g. star see http://getbootstrap.com/components/#glyphicons-glyphs")
+        help_text=_("Menu icon style name (else service default), e.g. star see https://getbootstrap.com/docs/3.4/components/#glyphicons")
```

Comment:
Yup, sorry about that. I had accidentally based this branch on the previous feature branch instead of master. And with the confusion I had regarding Seppo's branch and trying to squash the commits in my own branch, I didn't notice that one. But I feel like I've gotten better at handling branches and commit history since making this PR!